warrior/WarriorCore/testsuite_driver.py

```diff
@@ -335,14 +335,7 @@ def execute_testsuite(testsuite_filepath, data_repository, from_project,
                 break
     
     elif execution_type.upper() == 'RUN_MULTIPLE':
-        Max_Attempts = Utils.xml_Utils.getChildAttributebyParentTag(testsuite_filepath, 'Details', 'type', 'Max_Attempts')
-        Number_Attempts = Utils.xml_Utils.getChildAttributebyParentTag(testsuite_filepath, 'Details', 'type', 'Number_Attempts')
-        
-        if Max_Attempts == "":
-            execution_value = Number_Attempts
-        else:
-            execution_value = Max_Attempts
-        
+        execution_value = Utils.xml_Utils.getChildAttributebyParentTag(testsuite_filepath, 'Details', 'type', 'Number_Attempts')
```

Comment:
please add a check here to see if execution value is a valid integer otherwise throw a warning and default it to 1 so that the test runs only once